repobee_sanitizer/_sanitize.py

```diff
@@ -34,6 +34,23 @@ def sanitize_file(file_abs_path: pathlib.Path) -> Optional[str]:
         return "\n".join(sanitized_string)
 
 
+def strip_file(file_abs_path: pathlib.Path) -> Optional[str]:
```

Comment:
I've put in a bunch of comments on this function to point out some errors, but I think that ultimately this function should be removed entirely (see comments on `_strip`)